From c7263a0d3bc83b010b7ae8fb06c061303280babe Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Wed, 28 Feb 2024 18:11:58 +0100
Subject: [PATCH 0691/1695] plat-stm32mp1: default enable hardware RNG for
 STM32MP15 series

Default enable hardware RNG for STM32MP15x boards. Boards that don't
use a hardware RNG in OP-TEE should be part of flavorlist-no_rng.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 9ead80d6f294422b296e1a7fc0e174026df238dd plat-stm32mp1: default enable hardware RNG for STM32MP15 series]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I35655ee3d4e790239d918dc00bcc80cf76176907
---
 core/arch/arm/plat-stm32mp1/conf.mk | 1 -
 1 file changed, 1 deletion(-)

diff --git a/core/arch/arm/plat-stm32mp1/conf.mk b/core/arch/arm/plat-stm32mp1/conf.mk
index 9dafaba6c..ccc3105df 100644
--- a/core/arch/arm/plat-stm32mp1/conf.mk
+++ b/core/arch/arm/plat-stm32mp1/conf.mk
@@ -205,7 +205,6 @@ CFG_CORE_RESERVED_SHM ?= n
 CFG_HALT_CORES_ON_PANIC ?= y
 CFG_STM32_BSEC_SIP ?= y
 CFG_TEE_CORE_NB_CORE ?= 2
-CFG_WITH_SOFTWARE_PRNG ?= y
 CFG_STM32MP1_OPTEE_IN_SYSRAM ?= n
 ifeq ($(CFG_STM32MP1_OPTEE_IN_SYSRAM),y)
 $(call force,CFG_WITH_PAGER,y)
-- 
2.47.2

